mautic/mautic
# Fix incorrect grammar on password reset flash message

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✅ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

A very simple grammar fix which makes me twitch every time I see it! 'if user exist' is grammatically incorrect, it should be 'if user exists'.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try to send a password reset request and make sure it's showing 'exists' in the flash message. Also check the screen where you reset the password and make sure the success message is also updated to say 'exists'.
3. Make sure the tests are passing as it's used in a functional test.
